src/main/java/com/svydovets/bibirnate/annotation/OneToMany.java

```diff
@@ -0,0 +1,11 @@
+package com.svydovets.bibirnate.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface OneToMany {
```

Comment:
Please add java docs

Reply:
I would suggest leaving this without java docs as for now cause we have appropriate tickets for OneToMany and ManyToOne relations and I think that this annotation will be further extended

Reply:
let's add todo please instead.

Reply:
done